ddp-workspace/projects/ddp-singular/src/app/components/activity/activity.component.ts

```diff
@@ -15,6 +15,14 @@ export class ActivityComponent extends ActivityRedesignedComponent {
   Route = Route;
   isCaptchaResolved = false;
 
+  @HostListener('document:keydown.enter', ['$event']) onEnter(event:KeyboardEvent) {
+      super.nextField(event, {
+          elements: ['input', 'mat-select', 'select', 'textarea'],
+          preventDefault: true,
+          carousel: true
+      })
+  }
+
   get isPrequal(): boolean {
     return this.model && this.model.activityCode === ActivityCode.Prequal;
```

Comment:
**_#minor_**

can be shortened like 
`return this.model?.activityCode === ActivityCode.Prequal;`